examples/helm/205_clean_commerce.yaml

```diff
@@ -49,24 +50,10 @@ jobs:
     kind: "vtctlclient"
     command: "ApplyRoutingRules -rules=\'{}\'"
 
-vtctld:
-  serviceType: "NodePort"
-  resources:
-
-vtgate:
-  serviceType: "NodePort"
-  resources:
-
 vttablet:
-  mysqlSize: "prod"
-  resources:
-  mysqlResources:
   # It's generally not recommended to override this value for production usage.
```

Comment:
This comment line should probably get removed from all these examples because it's misleading